projects/plugins/boost/tests/e2e/lib/pages/wp-admin/JetpackBoostPage.js

```diff
@@ -22,6 +22,7 @@ export default class JetpackBoostPage extends WpPage {
 		const button = await this.page.$( '.jb-connection button' );
 		await button.click();
 		await this.waitForApiResponse( 'connection' );
+		await this.waitForApiResponse( 'optimizations/status' );
```

Comment:
Thanks for the suggestion. While that looks like the correct way, I am currently relying on Github actions to test the changes since my local setup is not working. I have to debug why my local e2e scripts are not running. I may not be able to prioritize this until our next release in a few days. I have created an issue accordingly: #24648

Reply:
LGTM!